python/metatensor-torch/metatensor/torch/atomistic/model.py

```diff
@@ -257,6 +257,18 @@ class MetatensorAtomisticModel(torch.nn.Module):
     >>> with tempfile.TemporaryDirectory() as directory:
     ...     model.save(os.path.join(directory, "constant-energy-model.pt"))
 
+    Recreating a full atomistic model from a saved model is also possible using
+    :py:func:`load_atomistic_model`:
+
+    >>> with tempfile.TemporaryDirectory() as directory:
+    ...     model.save(os.path.join(directory, "constant-energy-model.pt"))
+    ...     model_loaded = load_atomistic_model(
+    ...         os.path.join(directory, "constant-energy-model.pt")
+    ...     )
+    >>> model_recreated = MetatensorAtomisticModel(
+    ...     model_loaded, model_loaded.metadata(), model_loaded.capabilities()
+    ... )
+
```

Comment:
should we automatically do this in `load_atomistic_model`? I think it would make sense and remove the weird boilerplate

Reply:
yes! Good idea.